src/Register/Register.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import Card from '@material-ui/core/Card';
+
+class Register extends React.Component {
```

Comment:
Please change this to a functional component, and rename the file to `src/Register/index.js`